database/factories/PageFactory.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace Database\Factories\EscolaLms\Pages\Models;
+
+use Database\Factories\EscolaLms\Core\Models\UserFactory;
+use EscolaLms\Pages\Models\Page;
+use Illuminate\Database\Eloquent\Factories\Factory;
+use Illuminate\Support\Str;
+
+class PageFactory extends Factory
+{
+    protected $model = Page::class;
+
+    public function definition()
+    {
+        $title = $this->faker->catchPhrase;
+        return [
+            'slug' => Str::slug($title, '-'),
+            'title' => $title,
+            'author_id' => UserFactory::new(),
+            'content' => $this->faker->randomHtml(),
```

Comment:
replace with `markdown` please

Reply:
It's just a sample string, but if it makes a difference I've used a plugin for this. Is it ok?